job_template.sh

```diff
@@ -0,0 +1,11 @@
+#SBATCH --account=def-sabeda
+#SBATCH --time=0-08:00        # time (DD-HH:MM) |TODO to change
```

Comment:
@jcohenadad I wasn't sure of what *time* corresponds to in this case, is it for processing of a sub-dataset only?

Reply:
corresponds to the max allocation time on compute canada for the job (see [example](https://docs.computecanada.ca/wiki/Niagara_Quickstart#Example_submission_script_.28OpenMP.29)). And yes, only for sub-dataset

Reply:
OK thanks!